### PR TITLE
fix workerCountRatio when scaling down to 1 worker

### PR DESCRIPTION
**Pull request by fgimenez**
This will prevent errors when scaling down to 1 worker:
```
Invalid Autoscaling rolling update policy: MaxBatchSize must be greater than or equal to 1
```

### CODE REVIEW

File: service/awsconfig/v6/resource/cloudformation/adapter/auto_scaling_group.go [hunk 1]
```diff
@@ -41,5 +41,9 @@ func workerCountRatio(workers int, ratio float32) string {
 	value := float32(workers) * ratio
 	rounded := int(value + 0.5)
 
+	if workers > 0 && rounded == 0 {
+		rounded = 1
+	}
```
Comment (xh3b4sd) [inline service/awsconfig/v6/resource/cloudformation/adapter/auto_scaling_group.go]: v6 got just frozen. We need to introduce a new version bundle.

Comment (rossf7) [reply service/awsconfig/v6/resource/cloudformation/adapter/auto_scaling_group.go]: For me it makes sense to include this in v6 since its also a bug fix.

Comment (xh3b4sd) [reply service/awsconfig/v6/resource/cloudformation/adapter/auto_scaling_group.go]: Oh so it is not yet frozen?

Comment (xh3b4sd) [reply service/awsconfig/v6/resource/cloudformation/adapter/auto_scaling_group.go]: We should then also add another changelog. 

When do you plan to freeze this here then?

Comment (rossf7) [reply service/awsconfig/v6/resource/cloudformation/adapter/auto_scaling_group.go]: I'll add to the changelog. This will be the final change.

**Comment by fgimenez** [comment]
Thx! waiting for e2e

**Pull request merged**